packages/core/src/html/linkProcessor.js

```diff
@@ -148,7 +154,10 @@ function isValidFileAsset(resourcePath, config) {
  * @returns {string} these string return values are for unit testing purposes only
  */
 function validateIntraLink(resourcePath, cwf, config) {
-  if (!resourcePath || urlUtil.isUrl(resourcePath) || resourcePath.startsWith('#')) {
+  if (!resourcePath
+    || urlUtil.isUrl(resourcePath)
+    || resourcePath.startsWith('#')
+    || /^(?:mailto:|tel:)/i.test(resourcePath)) {
```

Comment:
Another possible change would be extracting all the `/^(?:mailto:|tel:)/i.test(resourcePath)` to a helper function in linkProcessor, though I'm not sure if that would be overcomplicating. Personally I think either way is fine!